forecasters/aardvark/R/get_forecasters.R

```diff
@@ -33,7 +33,7 @@
 #' ahead = 1
 #' my_forecaster <- get_forecasters(response = "jhu-csse_deaths_incidence_num",
 #'                                  ahead = ahead)[["aardvark_state_death_forecaster"]][["forecaster"]]
-# @importFrom evalforecast multinomial_preprocesser
+# @importFrom evalcast multinomial_preprocesser
```

Comment:
Oh I see.  I brought up my comment because of this line.

Reply:
Hmmm. It's not propagating to the NAMESPACE. That line is missing a ' after the hashtag, so roxygen ignored it. I'm guessing it's supposed to be a comment (I don't see it actually used anywhere). Probably best to delete to avoid confusion, but maybe Collin can tell if it's needed (and there's a mistake elsewhere) or not.

Reply:
oh didn't see the missing '.  Good point.

Reply:
It's not needed. I just left it in there as a comment until now because there has been some ambiguity about where the multinomial preprocessor will live.